internal/backup_fetch_handler_new.go

```diff
@@ -7,6 +7,61 @@ import (
 	"github.com/wal-g/wal-g/utility"
 )
 
+type FetchConfig struct {
+	filesToUnwrap   map[string]bool
+	// missingBlocks stores count of blocks missing for file path
+	missingBlocks   map[string]int64
+	tablespaceSpec  *TablespaceSpec
+	backupName      string
+	folder          storage.Folder
+	dbDataDirectory string
+}
+
+func (fc *FetchConfig) applyUnwrapResult(unwrapResult *UnwrapResult) {
```

Comment:
When I read this function I tried to find where you applied unwrapresult to real files/bytes))
You just set a fetch config with Unwrap Result here.

Reply:
Fixed in 1e027201cd3cc2fd810ac9a08404652d7d64fcef